diff --git a/node_modules/@googlemaps/react-native-navigation-sdk/android/build.gradle b/node_modules/@googlemaps/react-native-navigation-sdk/android/build.gradle
index 49426d7..9840e0b 100644
--- a/node_modules/@googlemaps/react-native-navigation-sdk/android/build.gradle
+++ b/node_modules/@googlemaps/react-native-navigation-sdk/android/build.gradle
@@ -59,6 +59,6 @@ dependencies {
   implementation "androidx.car.app:app:1.4.0"
   implementation "androidx.car.app:app-projected:1.4.0"
   implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
-  implementation "com.google.android.libraries.navigation:navigation:7.6.1"
+  implementation "com.google.android.libraries.navigation:navigation:7.9.0"
   api 'com.google.guava:guava:31.0.1-android'
 }
